UndertaleModTool/Scripts/Community Scripts/ImportGMS2FontData.csx

```diff
@@ -52,13 +52,18 @@ Try renaming the correct texture file to
 and putting it in the same directory as the .yy file."
     );
 
+bool tginExists = Data.TextureGroupInfo != null;
+// Default to putting the font into the default texgroup
+UndertaleTextureGroupInfo fontTexGroup;
+if (tginExists)
+    fontTexGroup = Data.TextureGroupInfo.ByName("Default");
 /*
-  If true, the script will attempt to add the new font (if any) and the new font glyph texture that it created to a texture group
-  This was an attempt to get fonts that this script creates appear in a specific 2022.3 game, but was proved unnecessary, as the problem was caused by something else
+  If true, the script will attempt to add the new font (if any) and the new font glyph texture
+  that it created to a texture group
+  This was an attempt to get fonts that this script creates appear in a specific 2022.3 game,
+  but it was proved unnecessary as the problem was caused by something else
 */
-bool attemptToFixFontNotAppearing = false; // Data.GM2022_3;
-// Default to putting the font into the default texgroup
-UndertaleTextureGroupInfo fontTexGroup = Data.TextureGroupInfo.ByName("Default");
+bool attemptToFixFontNotAppearing = tginExists && false; // Data.GM2022_3;
```

Comment:
why was the `tginExists` added to here?

Reply:
:thinking: Should I remove

Reply:
Ah, was not aware as in the previous iteration it didnt check for it. feel free to keep it then

Reply:
I decide to keep it